File: limesz_app/limesz_data/Models/LicenseInfo.cs
using System.ComponentModel.DataAnnotations;

namespace margarita_data.Models
{
    public class LicenseInfo: BaseRootModel
    {
        [Required]
        public string RestaurantId { get; set; }
        [Required]
        public LicenseType LicenseType { get; set; }
        public DateTime GrantedDate { get; set; }
        public DateTime ActivatedDate { get; set; }
        public DateTime ValidUntil { get; set; }
    }
}

File: limesz_client/projects/pluto/src/lib/api-providers/generated-api/models/license-info.ts
/* tslint:disable */
/* eslint-disable */
import { LicenseType } from '../models/license-type';
export interface LicenseInfo {
  activatedDate?: string;
  grantedDate?: string;
  id?: string | null;
  licenseType: LicenseType;
  restaurantId: string;
  validUntil?: string;
}
